rclcpp/src/rclcpp/generic_subscription.cpp

```diff
@@ -72,4 +71,55 @@ void GenericSubscription::return_serialized_message(
   message.reset();
 }
 
+
+// DYNAMIC TYPE ====================================================================================
+// TODO(methylDragon): Reorder later
+rclcpp::dynamic_typesupport::DynamicMessageType::SharedPtr
```

Comment:
@methylDragon super nit, but the functions above this grouping all have return types on the same line as the function (which is usually not what I see in this repo).

That said, I'm not requesting any changes here... Just pointing out that I found it odd.